basic-algorithms-&-datastructures/index040.js

```diff
@@ -0,0 +1,33 @@
+const newsFeed = {
+    page: "front_page",
+    articles: [{
+        articles:  [
+            {
+              headline: null,
+              image: [],
+              date_published: null,
+              date_modified: null,
+              author: {
+                username: null,
+                name: null
+              },
+              likes: null,
+              dislikes: null,
+              shares: null,
+              comments: null,
+              section: {
+                name: null,
+                subscribers: null
+              },
+              url: null
+        },{},{},{},{},{}
+    ]}
+]}
+
+/*
```

Comment:
/*
Set the value of `username` in the nested `author` object to the string "PeoplePerson" and the `name` key to the string "Pat Comb".
*/